bootstrap/terraform/main.tf

```diff
@@ -548,6 +544,8 @@ module "vpc" {
   source  = "terraform-aws-modules/vpc/aws"
   version = "~> 5.0"
 
+  manage_default_vpc = true
```

Comment:
what does `manage_default_vpc` do?

Reply:
The manage_default_vpc behaves differently from normal resources in that if a default VPC exists, Terraform does not create this resource, but instead "adopts" it into management. If no default VPC exists, Terraform creates a new default VPC, which leads to the implicit creation of [other resources](https://docs.aws.amazon.com/vpc/latest/userguide/default-vpc.html#default-vpc-components). By default, terraform destroy does not delete the default VPC but does remove the resource from Terraform state. 

It improves the create/destroy process. I've seen EKS Blueprints patterns using it.